Anlab.Mvc/Controllers/OrderController.cs

```diff
@@ -51,11 +51,20 @@ public OrderController(ApplicationDbContext context, IOrderService orderService,
                 InternalProcessingFee = Math.Ceiling(proc.Cost),
                 ExternalProcessingFee = Math.Ceiling(proc.Cost * _appSettings.NonUcRate)
             };
-
+            
             var user = _context.Users.Single(a => a.Id == CurrentUserId);
             model.DefaultAccount = user.Account?.ToUpper();
             model.DefaultEmail = user.Email;
 
+            var defaults = await _labworksService.GetClientDefaults(user.Email);

```

Comment:
@jSylvestre we should probably only get the defaults if the user profile has a clientID set